macOS/DuckDuckGo/PinnedTabs/View/PinnedTabView.swift

```diff
@@ -256,27 +258,72 @@ struct PinnedTabInnerView: View {
         }
     }
 
+    @ViewBuilder
+    var accessoryButton: some View {
+        if tabCrashIndicatorModel.isShowingIndicator {
+            crashIndicatorView
+        } else {
+            audioStateView
+        }
+    }
+
     private func audioIndicator(isMuted: Bool) -> some View {
-        ZStack {
-            Circle()
-                .stroke(Color.gray.opacity(0.5), lineWidth: 0.5)
-                .background(Circle().foregroundColor(.pinnedTabMuteStateCircle))
-                .frame(width: 16, height: 16)
+        Button {
+            model.muteUnmuteTab()
+        } label: {
+            ZStack {
+                Circle()
+                    .stroke(Color.gray.opacity(0.5), lineWidth: 0.5)
+                    .background(Circle().foregroundColor(.pinnedTabMuteStateCircle))
+                    .frame(width: 16, height: 16)
+
+                if isMuted {
+                    Image(.audioMute)
+                        .resizable()
+                        .frame(width: 12, height: 12)
+                } else {
+                    Image(.audio)
+                        .resizable()
+                        .frame(width: 12, height: 12)
+                }
+            }
+        }
+        .buttonStyle(.plain)
+        .offset(x: 8, y: -8)
+    }
 
-            if isMuted {
-                Image(.audioMute)
-                    .resizable()
-                    .frame(width: 12, height: 12)
-            } else {
-                Image(.audio)
+    @ViewBuilder
+    var crashIndicatorView: some View {
+        Button {
+            tabCrashIndicatorModel.isShowingPopover.toggle()
+        } label: {
+            ZStack {
+                Circle()
+                    .stroke(Color.gray.opacity(0.5), lineWidth: 0.5)
+                    .background(Circle().foregroundColor(.pinnedTabMuteStateCircle))
+                    .frame(width: 16, height: 16)
+
+                Image(.tabCrash)
                     .resizable()
                     .frame(width: 12, height: 12)
             }
         }
-        .offset(x: 8, y: -8)
-        .onTapGesture {
-            model.muteUnmuteTab()
+        .buttonStyle(.plain)
+        .popover(isPresented: $tabCrashIndicatorModel.isShowingPopover, arrowEdge: .bottom) {
+            PopoverMessageView(
+                viewModel: .init(
+                    title: "This tab has crashed",
```

Comment:
I know that we are putting this behind a feature flag, but then we will launch to all users. Are you going to add translations in a following PR?

Reply:
Yes, absolutely. I’ll add translations in the next PR, and also I’ll try to preemptively request translations based on Windows copy. I decided not to put these strings in UserText to limit noise related to untranslated UserText strings while this is in main waiting for translations.